include/ck/wrapper/operations/copy.hpp

```diff
@@ -37,5 +42,134 @@ __host__ __device__ void copy(const SrcTensorType& src_tensor, DstTensorType& ds
     }
 }
 
+/**
+ * \brief Perform optimized copy between two tensors. Tensors must have the
+ *  same size.
+ *
+ * \tparam DimAccessOrderTuple Tuple with dimension access order.
+ * \tparam VectorDim Dimension for vectorize read and write.
+ * \tparam ScalarPerVector Number of scalar per vectorize read and write.
```

Comment:
I would suggest adding like even 2-3 sentences for these points here. It's kinda clear for us since we know CK, but the API is mostly for people who do not want to deal with CK and all its tricks so imo the description is not particularly helpful here.
Maybe you could add some hints saying which dimensions should be used for vectorized read and writes, how to set number of scalar per vectorized read and so on? Also, "number of scalar" is not very descriptive in my opinion. Maybe "number of elements per single vectorized read" or sth like that?
```suggestion
 * \tparam DimAccessOrderTuple Tuple with dimension access order.
 * \tparam VectorDim Dimension for vectorized read and write.
 * \tparam ScalarPerVector Number of scalar per vectorized read and write.
```